avogadro/rendering/camera.h

```diff
@@ -121,6 +121,20 @@ class AVOGADRORENDERING_EXPORT Camera
    */
   void calculatePerspective(float fieldOfView, float zNear, float zFar);
 
+  /**
+   * << API Extension for TDX >>
```

Comment:
Please omit this. You may have added it, but there's plenty of reason to use this in other code.